project/AppDependencies.scala

```diff
@@ -3,25 +3,25 @@ import sbt.*
 
 object AppDependencies {
 
-  private val BootstrapPlayVersion = "8.3.0"
+  private val BootstrapPlayVersion = "8.5.0"
 
   val compile = Seq(
     "uk.gov.hmrc"   %% "bootstrap-backend-play-30" % BootstrapPlayVersion,
-    "org.typelevel" %% "cats-core"                 % "2.3.0",
+    "org.typelevel" %% "cats-core"                 % "2.10.0",
     "com.beachape"  %% "enumeratum-play-json"      % "1.8.0"
   )
 
   val test = Seq(
     "uk.gov.hmrc"                      %% "bootstrap-test-play-30"      % BootstrapPlayVersion            % Test,
-    "org.scalatest"                    %% "scalatest"                   % "3.2.5"                         % Test,
-    "org.scalamock"                    %% "scalamock"                   % "5.1.0"                         % Test,
+    "org.scalatest"                    %% "scalatest"                   % "3.2.18"                         % Test,
+    "org.scalamock"                    %% "scalamock"                   % "6.0.0"                         % Test,
```

Comment:
Maybe not upgrade scalamock now?
There is not conflict with how things are done but probably best to keep all projects symmetrical.

Reply:
Changed it back :)